### PR TITLE
Add iSH by @PseudonymPatel

### PR DESCRIPTION
**Pull request by PseudonymPatel**
2. [x] Update contents.json instead of README 
1. [x] Project URL:
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

**Comment by PseudonymPatel** [comment]
How would one go about fixing these errors? They are not part of the new app info I tried to commit

**Comment by vpeschenkov** [comment]
> How would one go about fixing these errors? They are not part of the new app info I tried to commit

Don't pay your attention to those errors. We'll fix them later.

File: contents.json [hunk 1]
```diff
@@ -329,6 +329,18 @@
...
   ],
   "projects": [
+        {
+                "title": "iSH",
+                "category-ids": ["developer", "emulator"],
+                "tags": ["swift"],
+                "description": "The Linux shell on iOS",
+                "source": "https://github.com/tbodt/ish",
+                "homepage": "https://ish.app",
+                "screenshots": ["https://imgur.com/a/sgC9ESK", "https://imgur.com/a/35rYIPn"],
```
Comment (vpeschenkov) [inline contents.json]: @PseudonymPatel these links should be direct to images.

File: contents.json [hunk 2]
```diff
@@ -329,6 +329,18 @@
     }
   ],
   "projects": [
+        {
+                "title": "iSH",
+                "category-ids": ["developer", "emulator"],
+                "tags": ["swift"],
```
Comment (vpeschenkov) [inline contents.json]: @PseudonymPatel I'm not sure about `swift`, the project is written mostly in `c` + `objc`. Let's use them instead.

**Comment by vpeschenkov** [comment]
@PseudonymPatel thank you!

**Pull request merged**